keps/sig-node/127-user-namespaces/README.md

```diff
@@ -603,6 +641,15 @@ use container runtime versions that have the needed changes.
 
 ### Upgrade / Downgrade Strategy
 
+Existing pods will still work as intended, as the new field is missing there.
+
+Upgrade will not change any current behaviors.
+
+When the new functionality wasn't yet used, downgrade will not be affected.
+
+Versions of Kubernetes that doesn't have this feature implemented will
+ignore and strip out the new field `pod.spec.hostUsers`.
```

Comment:
The object won't be modified, right? Reviews where picky on the words, so I would leave this to just "ignore the new field..."

Reply:
I'll fix it